### ISSUE TITLE
Can Claymore detect Ryzen 5 onboard graphics

### ISSUE THREAD

**Issue by ivierlyn** [nanopool/ClaymoreZECMiner#21]
Hi
I have the Ryzen 5 2400 and I wanted to know if it is possible or work being done to have Claymore recognize and take advantage of the built in Vega graphics on the Ryzen 5 APUs? I am running the latest version but it only detects my 560 and 580 cards at the moment. 

Thanks

**Comment by venadHD** [comment]
From when ryzen are APUs?

**Comment by venadHD** [comment]
2400G apu takes more energy than hashrate in global, thats optimized for games not mining, thats not HBM memory

**Comment by ivierlyn** [comment]
I understand the memory is not as high bandwidth but shouldn't it be possible to run some threads on each core even if the memory is not the most memory or do the algorithms really depend upon memory and I would have to split my mining on my machine between CPU and GPU to really take advantage of utilizing the hardware.

**Comment by venadHD** [comment]
Hello  ivierlyn

Take care about GPU on CPU chips are independent of CPU threads

Anyway, with other miners (zcash or other algo) are you able to mine?

**Comment by ivierlyn** [comment]
Yes my mining with my GPUs works great and I have no issues there. 

I was hoping that there would be some sort of work done to be able to get some more hashing out of the graphics on the CPU with the Vega on-board graphics but maybe that was just wishful thinking on my part :). Thanks for your time and answering my questions.

**Comment by venadHD** [comment]
Hello ivierlyn

i asked about if you with another algo or mining program you would able to
mine?
Are only this miner restricting your GPU onBoar mining? No dedicated GPUs
in this questions

**Comment by ivierlyn** [comment]
If I understand you are asking if any other miner allows me to use the integrated graphics, and so far I have not found one. I like using Claymore as I have AMD and this one works really well with the cards I have at the moment.

**Comment by venadHD** [comment]
Hello ivierlyn

So maybe its restricted by processor, but i really no knows in a 100% trust
